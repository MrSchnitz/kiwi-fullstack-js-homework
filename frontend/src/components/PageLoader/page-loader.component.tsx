import React from "react";
import {Loading} from "@kiwicom/orbit-components/lib";
import {PageLoaderLayout} from "./page-loader.styles";

const PageLoader: React.FC = () => {
  return (
    <PageLoaderLayout>
      <Loading type={"pageLoader"} />
    </PageLoaderLayout>
  );
};

export default PageLoader;
