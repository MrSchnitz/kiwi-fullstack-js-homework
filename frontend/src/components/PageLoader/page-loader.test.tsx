import React from "react";
import { shallow } from "enzyme";
import PageLoader from "./page-loader.component";

describe("PageLoader component", () => {
  let wrapper: any;
  beforeEach(() => {
    wrapper = shallow(<PageLoader />);
  });

  it("should render PageLoader component", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
