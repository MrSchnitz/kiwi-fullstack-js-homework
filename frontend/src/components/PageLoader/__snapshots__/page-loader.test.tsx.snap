// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageLoader component should render PageLoader component 1`] = `ShallowWrapper {}`;
